kits/daisy/CMakeLists.txt

```diff
@@ -0,0 +1,136 @@
+  ${CMAKE_CURRENT_SOURCE_DIR}/src/xml_util/xml_helpers.cpp
+)
+
+SET(PARAMETER_GUI_SOURCES
+  ${CMAKE_CURRENT_SOURCE_DIR}/src/parameter_gui.cpp
+  ${CMAKE_CURRENT_SOURCE_DIR}/src/param_display/hexapod_parameter_view.cpp
+  ${CMAKE_CURRENT_SOURCE_DIR}/src/param_display/hexapod_parameter_gui.cpp
+  ${CMAKE_CURRENT_SOURCE_DIR}/src/input/input_manager_mobile_io.cpp
+  ${CMAKE_CURRENT_SOURCE_DIR}/src/xml_util/pugixml.cpp
+  ${CMAKE_CURRENT_SOURCE_DIR}/src/xml_util/xml_helpers.cpp
+)
+
+SET(INPUT_TEST_SOURCES
+  ${CMAKE_CURRENT_SOURCE_DIR}/src/input_test.cpp
+  ${CMAKE_CURRENT_SOURCE_DIR}/src/input/input_manager_mobile_io.cpp
+  ${CMAKE_CURRENT_SOURCE_DIR}/src/xml_util/pugixml.cpp
+  ${CMAKE_CURRENT_SOURCE_DIR}/src/xml_util/xml_helpers.cpp
+)
+
+add_executable(hexapod_control ${SOURCES} $<TARGET_OBJECTS:hexapod_core>)
+qt5_use_modules(hexapod_control Core Gui Widgets)
+
+if (WIN32) 
+target_link_libraries( hexapod_control ${SONAME} kernel32 ) # kernel32 for sleep commands.
+else()
+target_link_libraries( hexapod_control ${SONAME} hebi hebic++ m pthread)
+endif()
+
+# Add ultra-conservative warnings.
+if ( CMAKE_COMPILER_IS_GNUCC )
+  set_property( TARGET hexapod_control APPEND_STRING PROPERTY COMPILE_FLAGS " -Wall " )
+endif ( CMAKE_COMPILER_IS_GNUCC )
+# For Windows, we copy the .dll file into the binary directory so that we don't have to set the PATH variable.
+if (WIN32)
+add_custom_command( TARGET hexapod_control POST_BUILD
+  COMMAND ${CMAKE_COMMAND} -E copy_if_different
+  "${PROJECT_SOURCE_DIR}/lib/${SONAME}.dll"
+  $<TARGET_FILE_DIR:hexapod_control>)
+endif()
+
+add_custom_command(TARGET hexapod_control POST_BUILD
+                   COMMAND ${CMAKE_COMMAND} -E copy_directory
+                       ${CMAKE_CURRENT_SOURCE_DIR}/resources/ $<TARGET_FILE_DIR:hexapod_control>/)
+
+# Create logs directory in case the example logs data.
+add_custom_command(TARGET hexapod_control POST_BUILD
+                   COMMAND ${CMAKE_COMMAND} -E make_directory
+                       $<TARGET_FILE_DIR:hexapod_control>/logs)
+
+# Build all examples:
+# Parameter setting GUI
+# NOTE: due to an issue with the Qt53DRender ubuntu 18.04 package, we have
+# temporarily disabled the 3D parameter GUI.  To reenable, add support for 3D Qt
+# components in the above `find_package` scripts, and uncomment the following
+# lines.
+#add_executable(parameter_gui ${PARAMETER_GUI_SOURCES} $<TARGET_OBJECTS:hexapod_core>)
+#qt5_use_modules(parameter_gui Core Gui Widgets 3DCore 3DExtras 3DRender 3DInput)
+#target_link_libraries(parameter_gui ${SONAME} hebi hebic++ m pthread)
+
+# 2D Parameter setting GUI
+add_executable(parameter_gui_2d ${PARAMETER_GUI_2D_SOURCES} $<TARGET_OBJECTS:hexapod_core>)
+qt5_use_modules(parameter_gui_2d Core Gui Widgets)
+target_link_libraries(parameter_gui_2d ${SONAME} hebi hebic++ m pthread)
+
+add_executable(input_test ${INPUT_TEST_SOURCES})
+target_link_libraries( input_test ${SONAME} hebi hebic++ m pthread )
```

Comment:
I unfortunately remember that this won't work on Windows, because Windows `hebic++` doesn't actually export any functions (no `__declspec(dllexport)` on anything due to obvious oversight on my part)

Reply:
This is basically a comment, so there's nothing to resolve here ATM